frontend/sass/styles.scss

```diff
@@ -28,6 +28,7 @@
 @import "./_helpers.scss";
 @import "./_language-toggle.scss";
 @import "./_email.scss";
+@import "./_accordion.scss";
```

Comment:
At present, this is only used by the style guide on app.justfix.nyc, so this here is a bit of style bloat in a way, but it's not a lot of bloat and it will make it super easy for us to use this component from that site in the future too.

Reply:
cool!